### DISCUSSION TITLE
Can't get the result in an acceptable time with the feasible model

### DISCUSSION THREAD

**Discussion by songtaowang-saa** [yalmip/YALMIP#1194] [General discussion]
dear, community! 
I have a question about too much time that yalmip spends to solves my optimization problem. First, I think the obj and constraints are right, and I make sure that the model is feasible by using gurobi_iis function. But when I use optimize() function to solve the problem, It takes more than 20 minutes and still doesn't have a optimal result. So I want to know the problem in my code that cause this situation.(The picture below shows the problem)
![image](https://user-images.githubusercontent.com/115172165/201555724-a3685147-c82d-4b16-a0ae-592d2a5dcb64.png)
![image](https://user-images.githubusercontent.com/115172165/201555323-34e70be5-768c-4bb2-b57f-c8f5358e9745.png)
the detail message in the process: 
-Presolved model has 21 quadratic constraint(s)
-Root relaxation presolved model has 21 second-order cone constraints
I think that the problem maybe relate to above message, but I don't know how to solve it.
whole message:
![image](https://user-images.githubusercontent.com/115172165/201555455-f01f1cd1-8444-48f1-ac1e-b1aa99ff1cbf.png)

**Comment by johanlofberg** [comment]
You have 100 sos2 constraints. Any reason to believe that it should be possible to solve the MISCOP-model efficiently?

**Reply by songtaowang-saa** [reply]
Sorry. Could you please give me a link that introdces the MISCOP_model. I have no idea about what is the MISCOP_model. Secondly, in the other code that I wrote, the result can be obtained in 5 seconds with the similar content and different data, and there is 188 sos1 constraints. if possiable, can you give me further explaination about the problem. I will be great thankful for your help.
![image](https://user-images.githubusercontent.com/115172165/201608842-a5e79bf1-62da-48fa-b20e-d67426e8cc7a.png)
![image](https://user-images.githubusercontent.com/115172165/201609028-1a48f331-22a8-46cb-ad3f-b57351a3394f.png)
![image](https://user-images.githubusercontent.com/115172165/201608968-97de2892-8b63-4747-989a-9fc071f8d66f.png)
